mtp_api/apps/security/serializers.py

```diff
@@ -219,19 +219,32 @@ def update(self, instance, validated_data):
 
 class CheckSerializer(serializers.ModelSerializer):
     credit = SecurityCreditSerializer()
+    full_name = serializers.SerializerMethodField('get_full_name_from_user')
 
     class Meta:
         model = Check
         fields = (
             'id',
             'credit',
+            'created',
```

Comment:
what are you thinking of using this for? it's not quite the same as a credit's `started_at`